examples/service-clients/azure-client/external-controller/src/app.ts

```diff
@@ -3,19 +3,23 @@
  * Licensed under the MIT License.
  */
 
-import {
-	acquirePresenceViaDataObject,
-	ExperimentalPresenceManager,
-} from "@fluid-experimental/presence";
 import {
 	AzureClient,
 	AzureContainerServices,
 	AzureLocalConnectionConfig,
 	AzureRemoteConnectionConfig,
 } from "@fluidframework/azure-client";
+// eslint-disable-next-line import/no-internal-modules
```

Comment:
Are these necessary? Can we update the package's eslint config to allow imports from "/internal"?

Reply:
Previous discussion about this: https://github.com/microsoft/FluidFramework/pull/23073#discussion_r1841961099 @jason-ha

Reply:
There are more than I was expecting of these and we should do something about that. I just wasn't going to push harder in this PR that needs to happen for Ignite. I accept the tech debt.